virttest/storage.py

```diff
@@ -811,9 +804,7 @@ def copy_data_remote(self, src, dst):
     def copy_data_raw(src, dst):
         """Using dd for raw device."""
         if os.path.exists(src):
-            _dst = dst + '.part'
-            process.system("dd if=%s of=%s bs=4k conv=sync" % (src, _dst))
-            os.rename(_dst, dst)
+            process.system("dd if=%s of=%s bs=4k conv=sync" % (src, dst))
```

Comment:
Well this `.part` procedure is very important and was actually introduced to prevent broken backups. In case of a failure, the backup fails and the next test keeps using the previously existing backup or simply fails as there is no finished backup available. This is better then attempting to boot a partial backup, isn't it?

Reply:
^^ after playing with it please see the main comment: https://github.com/avocado-framework/avocado-vt/pull/2485#issuecomment-607217983 and potentially disregard this message... I'm not using raw devices so not entirely sure about the magic around.